frontend/src/domains/admin/signUp/SignUp.tsx

```diff
@@ -0,0 +1,96 @@
+import BasicButton from '@/components/BasicButton/BasicButton';
+import { ROUTES } from '@/constants';
+import {
+  SignUpField,
+  signUpFields,
+} from '@/domains/admin/signUp/constants/signUpFields';
+import useConfirmPassword from '@/domains/admin/signUp/hooks/useConfirmPassword';
+import {
+  fieldContainer,
+  signUpCaptionContainer,
+  signUpForm,
+} from '@/domains/admin/signUp/SignUp.style';
+import AuthLayout from '@/domains/components/AuthLayout/AuthLayout';
+import FormField from '@/domains/components/FormField/FormField';
+import useAuthForm from '@/domains/hooks/useAuthForm';
+import { useAppTheme } from '@/hooks/useAppTheme';
+import {
+  validateId,
+  validateName,
+  validatePassword,
+} from '@/utils/authValidations';
+import { useNavigate } from 'react-router-dom';
+
+export default function SignUp() {
+  const theme = useAppTheme();
+  const navigate = useNavigate();
+
+  const signUpValidators = {
+    name: validateName,
+    id: validateId,
+    password: validatePassword,
+  };
+
+  const {
+    authValue: signUpValue,
+    handleChangeForm,
+    errors,
+  } = useAuthForm({
+    initValues: {
+      name: '',
+      id: '',
+      password: '',
+    },
+    validators: signUpValidators,
+  });
+
+  const {
+    authValue: confirmPasswordValue,
+    handleChangeConfirmPassword,
+    errors: confirmPasswordErrors,
+  } = useConfirmPassword({
+    initValues: { confirmPassword: '', password: signUpValue.password },
+  });
+
+  return (
+    <AuthLayout title='회원가입' caption='새로운 계정을 만들어보세요'>
+      <form css={signUpForm(theme)}>
+        <div css={fieldContainer}>
+          {signUpFields.map((field: SignUpField) => (
+            <FormField
+              type={field.type}
+              key={field.name}
+              id={field.name}
+              label={field.labelKey}
+              value={
+                field.name === 'passwordConfirm'
+                  ? confirmPasswordValue.confirmPassword
+                  : signUpValue[field.name]
+              }
+              onChange={
+                field.name === 'passwordConfirm'
+                  ? handleChangeConfirmPassword
+                  : handleChangeForm
+              }
+              maxLength={field.maxLength}
+              minLength={field.minLength}
+              placeholder={field.placeholder}
+              errorMessage={
+                field.name === 'passwordConfirm'
+                  ? confirmPasswordErrors || ''
+                  : errors[field.name] || ''
+              }
+            />
+          ))}
```

Comment:
```suggestion
        const isConfirm = field.name === 'passwordConfirm';
  
  const props = {
    value: isConfirm
      ? confirmPasswordValue.confirmPassword
      : signUpValue[field.name],
    onChange: isConfirm
      ? handleChangeConfirmPassword
      : handleChangeForm,
    errorMessage: isConfirm
      ? confirmPasswordErrors || ''
      : errors[field.name] || '',
  };

  return (
    <FormField
      key={field.name}
      type={field.type}
      id={field.name}
      label={field.labelKey}
      maxLength={field.maxLength}
      minLength={field.minLength}
      placeholder={field.placeholder}
      {...props}
    />
```

속성안에 똑같은 조건문이 많아서 위와 같이 수정하는건 어떤가요?? 보고 괜찮으면 수정하고 안괜찮으면 넘어가도 괜찮습니다~~